proto/attestation/endorsement.proto

```diff
@@ -30,6 +30,9 @@ message TransparentReleaseEndorsement {
 
   // The log entry as proof of inclusion of the endorsement statement in Rekor.
   bytes rekor_log_entry = 2;
+
+  // The signature for the endorsement.
```

Comment:
I was assuming that `endorsement` above would have the signature -- since it does not, could you please add a note to the comment above it, for clarity?

Reply:
Expanded the comment with the link to the JSON message format